recipes/cctz/all/conanfile.py

```diff
@@ -56,8 +58,9 @@ def generate(self):
         tc.variables["BUILD_BENCHMARK"] = False
         # For shared msvc
         tc.variables["CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS"] = True
-        # Relocatable shared lib on Macos
-        tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0042"] = "NEW"
+        tc.cache_variables["CMAKE_POLICY_VERSION_MINIMUM"] = "3.5" # CMake 4 support (master branch already has 3.16)
+        if Version(self.version) > "2.4": # pylint: disable=conan-unreachable-upper-version
```

Comment:
if upsteram already has fixed this and we have a realistic expectation that the newer version will be cut from their main branch (the change was added 8 months ago!) - then I wouldn't error out for a new version, just apply this to all versions 2.4 or lower